tests/foreman/ui/test_dashboard.py

```diff
@@ -0,0 +1,197 @@
+        6.Click Reset to default
+
+        @Assert: Dashboard is reset to default.
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_addwidgets(self):
+        """@Test: Add Widgets to Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Content Dashboard
+        2.Select Manage Dropdown box
+        3.Add Widgets
+
+        @Assert: User is able to add widgets.
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_bookmark_search(self):
+        """@Test: Bookmark the search filter in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Content Dashboard
+        2.Add a filter to search box (eg. environment)
+        3.Bookmark the search filter
+
+        @Assert: User is able to list the Bookmark
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_host_configuration_status(self):
+        """@Test: Check if Host Configuration Status
+        Widget links are working
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Content Dashboard
+        2.Focus on the Host Configuration Status
+        3.Select each of the link which
+        has search string associated with it
+
+        @Assert: Make sure each search takes you to correct info
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_host_configuration(self):
+        """@Test: Check if Host Configuration Chart
+        is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Content Dashboard
+        2.Focus on Host Configuration Chart
+        3.Select the Circular chart which
+        has search string associated with it
+
+        @Assert: Make sure search string takes you to correct info
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_task_status(self):
+        """@Test: Check if Task Status Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Content Dashboard
+        2.Focus on Task Status
+        3.Click each link
+
+        @Assert: Make sure search string takes you to correct info
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_warning_error_task_status(self):
+        """@Test: Check if Latest Warning/Error
+        Tasks Status are working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Content Dashboard
+        2.Focus on Latest Warning/Error Tasks
+
+        @Assert: The links to all failed/warnings tasks are working
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_contentview_history(self):
+        """@Test: Check if Content View History
+           are working in Dashboard UI
```

Comment:
widget is working in Dashboard UI (please change this in all occurences after this)